src/zugferd211/Model/AllowanceCharge.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+namespace Easybill\ZUGFeRD211\Model;
+
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * Class AllowanceCharge
+ *
+ * @package Easybill\ZUGFeRD\Model
+ */
+class AllowanceCharge
```

Comment:
Please change the name to `TradeAllowanceCharge` as it is written in the XSD files. The suffix 'Type' is not used. 

![image](https://user-images.githubusercontent.com/318564/99075453-cc32f080-25b9-11eb-8a95-67d535943965.png)

Reply:
I don't understand… Should I rename the class? In the `xsd` files e.g. [FACTUR-X_EN16931_…Entity_100.xsd](https://github.com/easybill/zugferd-php/blob/2890a319fc924dda43e14144429355c93297a181/src/zugferd211/Schema/EN16931/FACTUR-X_EN16931_urn_un_unece_uncefact_data_standard_ReusableAggregateBusinessInformationEntity_100.xsd) the complex types are always named with `Type` suffix, but your classes don't have this suffix <img width="771" src="https://user-images.githubusercontent.com/6592155/99078006-f4245300-25bd-11eb-84a8-250b8b25feb3.png">, e.g. [Amount](https://github.com/easybill/zugferd-php/blob/master/src/zugferd211/Model/Amount.php) class. 
If you do not mind, please explain it to me.

Reply:
@blackgwe I have added a suggestion. The suffiy `Type` we dont use.